File: Lexer/RightCurlyToken.cpp
#include "token.h"
#include <string>
using namespace std;

class RightCurl : public Token {
	bool Token::equals()
	{

	}

	int Token::hashCode()
	{
		return 6;
	}

	string Token::toString()
	{
		return "}";
	}
};

File: src/main/java/parser/RightCurlyToken.java
package parser;

public class RightCurlyToken implements Token{
	public boolean equals(final Object other) {
		return other instanceof RightCurlyToken;
	}
	
	public int hashCode() {
		return 9;
	}
	
	public String toString() {
		return "RightCurlyToken";
	}

}
